### PR TITLE
Remove unused backoffice footer

### PR DESCRIPTION
**Pull request by Progi1984**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Remove unused backoffice footer (and call to external JS) : After inspection, the footer was hidden and never displayed (check in JS files). We kept a small part which contains hook `displayBackOfficeFooter`.
| Type?             | refacto
| Category?         | BO
| BC breaks?        | yes
| Deprecations?     | no
| Fixed ticket?     | N/A
| How to test?      | The footer in legacy page contains only hook `displayBackOfficeFooter`

**:notebook: BC Breaks :**
* Removed global variable `$timer_start`
* Remove public attribute `$timer_start` in `classes/controller/AdminController.php`

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/24786)

### CODE REVIEW

**Review by atomiix — DISMISSED** [review]

File: admin-dev/themes/default/template/footer.tpl [hunk 1]
```diff
@@ -27,85 +27,9 @@
 </div>
 {if $display_footer}
 <div id="footer" class="bootstrap hide">
-
-	<div class="col-sm-2 hidden-xs">
-		<a href="https://www.prestashop.com/" target="_blank" rel="noopener noreferrer nofollow">PrestaShop&trade;</a>
-		-
-		<span id="footer-load-time"><i class="icon-time" title="{l s='Load time:' d='Admin.Navigation.Footer'}"></i> {number_format(microtime(true) - $timer_start, 3, '.', '')}s</span>
```
Comment (matks) [inline admin-dev/themes/default/template/footer.tpl:34]: It's possible we can also remove `$timer_start`

File: admin-dev/themes/default/template/footer.tpl [hunk 2]
```diff
@@ -27,85 +27,9 @@
...
-			{if $host_mode}
-			/&nbsp;
-			<a href="https://status.prestashop.com/" class="footer_link _blank">
-				<i class="icon-circle status-page-dot"></i>
-				<span class="status-page-description"></span>
-			</a>
-			{/if}
-			{if $iso_is_fr && !$host_mode}
-			<p>Questions • Renseignements • Formations :
-				<strong>+33 (0)1.40.18.30.04</strong>
```
Comment (matks) [inline admin-dev/themes/default/template/footer.tpl:98]: 🤯

**Review by matks — DISMISSED** [review]
@Progi1984 nice !

I added one suggestion, not blocking. So I approve the PR.

Would it be possible to provide a PR description a little more complete to allow every GitHub visitor to understand what is this "unused backoffice footer"? A screenshot would be nice.

If this PR was coming from someone else than you, I would be very suspicious 😄  . I know you and I have the context but writing the context inside the PR body would be awesome.

**Review by NeOMakinG — DISMISSED** [review]

File: admin-dev/themes/default/template/footer.tpl (hunk 1 above)
Comment (Progi1984) [reply admin-dev/themes/default/template/footer.tpl:34]: Done & BC Break explained

**Comment by Progi1984** [comment]
> @Progi1984 nice !
> 
> I added one suggestion, not blocking. So I approve the PR.
> 
> Would it be possible to provide a PR description a little more complete to allow every GitHub visitor to understand what is this "unused backoffice footer"? A screenshot would be nice.
> 
> If this PR was coming from someone else than you, I would be very suspicious smile . I know you and I have the context but writing the context inside the PR body would be awesome.

Done

**Comment by Julievrz** [comment]
I removed the "waiting for wording" label as there is no wording in this PR but feel free to put it back if needed. :-)

**Review by matks — DISMISSED** [review]

**Review by matthieu-rolland — DISMISSED** [review]

File: admin-dev/themes/default/template/footer.tpl [hunk 3]
```diff
@@ -27,85 +27,9 @@
 </div>
 {if $display_footer}
 <div id="footer" class="bootstrap hide">
```
Comment (NeOMakinG) [inline admin-dev/themes/default/template/footer.tpl]:
```suggestion
<div id="footer" class="bootstrap">
```

File: admin-dev/themes/default/template/footer.tpl [hunk 4]
```diff
@@ -27,85 +27,9 @@
...
-			{/if}
-			{if $iso_is_fr && !$host_mode}
-			<p>Questions • Renseignements • Formations :
-				<strong>+33 (0)1.40.18.30.04</strong>
-			</p>
-			{/if}
-		</div>
-	</div>
-
 	<div class="col-sm-3">
```
Comment (NeOMakinG) [inline admin-dev/themes/default/template/footer.tpl]:
```suggestion
	<div class="col-sm-12">
```

But it's even useless, the entire div could be removed :)

**Comment by NeOMakinG** [comment]
Footer needs `padding: 105px 10px 0 225px;` overwise the menu is overflowing it

And it should be removed when the screens hit < 1038px

**Review by NeOMakinG — DISMISSED** [review]
Notified some problems, don't hesitate if you need help

**Review by matks — APPROVED** [review]

**Review by PierreRambaud — APPROVED** [review]

**Comment by matks** [comment]
Thank you @Progi1984

**Pull request merged**